### PR TITLE
Fix missing "star" icon in Cura 4.11

### PR DESCRIPTION
**Pull request by fieldOfView**
This PR adds an alias for the previously existing "star" icon that was used in the header for the profile dropdown to show a profile had changed settings.

The PR is against version 4.11, because if the alias is not there in Cura 4.11 then it makes little sense to add it back to the next version of Cura.

### CODE REVIEW

**Review by nallath — APPROVED** [review]

**Comment by nallath** [comment]
Whoops. We missed that. 

I will wait for CI to complete before merging it.

**Comment by fieldOfView** [comment]
"cross2" is also missing by the way. I'll see if I can find more before I open another PR.

**Comment by fieldOfView** [comment]
All of these are missing:
```
    basic
    buildplate
    category_shield
    category_unknown
    cross2
    drop_down_button
    group
    link
    play
    print_time
    printer_cloud_connected
    printer_cloud_not_available
    printer_connected
    printsetup
    quick
    scale_max
    setting_per_object
    star_empty
    star_filled
    ulti
    view_xray
    viewMode
```
That's so many that it does not look like it happened on accident. Is there a reasoning behind which icons were included and which icons were left out?

**Comment by nallath** [comment]
Those were left out on accident. We manually went over the changes, but we just missed out a whole bunch.

The idea was to have a full list so that we would have better backwards compatibility.

**Pull request merged**
